### ISSUE TITLE
Backport CCMA performance improvements

### ISSUE THREAD

**Issue by jchodera** [FoldingAtHome/openmm#31]
The changes in https://github.com/openmm/openmm/pull/2818 would also be super helpful for the COVID Moonshot.

**Comment by peastman** [comment]
I don't think that will be practical.  That code is very different between 7.4 and 7.5, since it got completely overhauled while unifying the CUDA and OpenCL implementations.  It will be a better use of our time to just focus on getting out a 7.5 based core.
